lib/compose-tweet.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = (commit, standard) => {
+  let [commitTitle] = commit.message.split("\n");
+  for (const exclude of ["Editorial", "Meta", "Review Draft Publication"]) {
+    if (commitTitle.startsWith(`${exclude}:`)) {
+      return null;
+    }
+  }
+
+  const authorEmails = standard.authors.map(author => author.email);
+  if (!authorEmails.includes(commit.author.email)) {
+    commitTitle += ` (thanks ${commit.author.name}!)`;
```

Comment:
We could also use author username, which might be shorter and more familiar. Dunno.